site/components/YoutubeEmbed/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, HTMLAttributes, useState } from "react";
+import { toNumber } from "lodash";
+import * as styles from "./styles";
+import Image, { ImageProps } from "next/image";
+
+/**
+ * For more information about the available youtube embed parameters see:
+ * https://developers.google.com/youtube/player_parameters#release_notes_08_23_2018
+ */
+type YoutubeEmbedProps = {
+  videoId: string;
+  autoplay?: boolean;
+  showSubtitles?: boolean;
+  hideControls?: boolean;
+  hideFullscreen?: boolean;
+  posterImg?: {
+    src: ImageProps["src"];
+    alt: string;
+  };
+} & HTMLAttributes<HTMLDivElement>;
```

Comment:
no need for this or the `...props` spread on L50, if we do ever need a `div` attribute we can be explicit about which ones the component supports

Reply:
Updated 👍